src/test/java/com/salesforce/mirus/KafkaMonitorTest.java

```diff
@@ -138,4 +140,115 @@ public void shouldNotReBalanceIfOnlySourcePartitionOrderHasChanged() {
             new PartitionInfo("topic1", 0, null, null, null)));
     assertThat(kafkaMonitor.partitionsChanged(), is(false));
   }
+
+  @Test
+  public void shouldApplyTopicRenameTransforms() {
+    Map<String, String> properties = new HashMap<>();
```

Comment:
There's a fair bit of duplication in the setup for these tests, recommendations on reducing that appreciated.

Reply:
It looks like you could create two helper methods to contribute prefix/suffix config and regex rename config respectively to the properties instance. You'd have to use the values from the test that contains both; but those, I think, would work independently. Then just call `addPrefixSuffixConfig` or `addRegexTransformConfig` as needed in your test cases.

I think the rest of the stuff is hard to refactor away in a way that wouldn't obscure the workings of the test.